docs/ref/django-admin.txt

```diff
@@ -1850,18 +1850,18 @@ Extra niceties
 Syntax coloring
 ---------------
 
+.. versionchanged:: 3.2
+
+  Added enabled-by-default support for colored terminal output on Windows via
+  colorama_.
+
```

Comment:
I think we can drop these in favour of the install instructions (which link here.)